src/test/java/com/cag/cagbackendapi/controllers/ProfileControllerTest.kt

```diff
@@ -33,7 +33,7 @@ class ProfileControllerTest {
         val userIdUUID = UUID.randomUUID()
         val userId = userIdUUID.toString()
         val userProfile = ProfileRegistrationDto(pronouns = "he/him", lgbtqplus_member = false, gender_identity = "", comfortable_playing_transition = true, comfortable_playing_man = true, comfortable_playing_women = true, comfortable_playing_neither = false, height_inches = 88, agency = "Pedro LLC", website_link_one = "", website_link_two = "", website_type_one = "", website_type_two = "", bio = "this is my bio")
-        val userProfileResponse = ProfileDto(userIdUUID, pronouns = "he/him", lgbtqplus_member = false, gender_identity = "", comfortable_playing_transition = true, comfortable_playing_man = true, comfortable_playing_women = true, comfortable_playing_neither = false, height_inches = 88, agency = "Pedro LLC", website_link_one = "", website_link_two = "", website_type_one = "", website_type_two = "", bio = "this is my bio")
+        val userProfileResponse = ProfileDto(userIdUUID, pronouns = "he/him", lgbtqplus_member = false, gender_identity = "", comfortable_playing_transition = true, comfortable_playing_man = true, comfortable_playing_women = true, comfortable_playing_neither = false, height_inches = 88, agency = "Pedro LLC", website_link_one = "", website_link_two = "", website_type_one = "", website_type_two = "", bio = "this is my bio", userEntity = null)
```

Comment:
Setting default values in the DTOs does not make it mandatory that a request must have that field. On our Swagger page, it will display the schema of the DTO with the user entity, but it is absolutely not required to send it. This will likely confuse the Frontend devs when they view our swagger, we'll have to inform them or make a separate DTO without the userEntity attached.